docs/how-to-guides/hello-world-example.md

```diff
@@ -65,73 +65,58 @@ FROM nginx:alpine
 ```
 
-**Build and run the image for verification purposes:**
+**Build Image from code locally:**
+
+:warning: **Note:** Please note that you need to pass the flag `--namespace k8s.io` to the `nerdctl` build command, so that `nerdctl` builds the image and then makes it available in the `k8s.io` namespace.
 
 <Tabs groupId="container-runtime">
   <TabItem value="nerdctl" default>
 
 ```
-nerdctl build -t nginx-helloworld:v1.0 .
-nerdctl images | grep nginx-helloworld
-nerdctl run -d -p 8080:80 --name my-site nginx-helloworld:v1.0
+nerdctl build --namespace k8s.io -t nginx-helloworld:latest .
+nerdctl images --namespace k8s.io | grep nginx-helloworld
 ```
 
   </TabItem>
   <TabItem value="docker">
 
 ```
-docker build -t nginx-helloworld:v1.0 .
+docker build -t nginx-helloworld:latest .
 docker images | grep nginx-helloworld
-docker run -d -p 8080:80 --name my-site nginx-helloworld:v1.0
 ```
   </TabItem>
 </Tabs>
 
-**Check your list of running containers:**
-
-<Tabs groupId="container-runtime">
-  <TabItem value="nerdctl" default>
+**Deploy to Kubernetes**
 
-```
-nerdctl ps
-CONTAINER ID    IMAGE                                      COMMAND                   CREATED           STATUS    PORTS                   NAMES
-b6775a69ab93    docker.io/library/nginx-helloworld:v1.0    "/docker-entrypoint.…"    48 seconds ago    Up        0.0.0.0:8080->80/tcp    my-site
-```
+Run below command to create and run a pod using the image built in the previous step. 
 
-  </TabItem>
-  <TabItem value="docker">
+:warning: **Note:** Please note that you need to pass the flag `--image-pull-policy=Never` to use a local image with `:latest` tag, as `:latest` tag will always try to pull the images from a remote repository.
 
 ```
-docker ps 
-CONTAINER ID   IMAGE                   COMMAND                  CREATED          STATUS          PORTS                                   NAMES
-5f50ba3f504c   nginx-helloworld:v1.0   "/docker-entrypoint.…"   7 seconds ago    Up 6 seconds    0.0.0.0:8080->80/tcp, :::8080->80/tcp   my-site
+kubectl run hello-world --image=nginx-helloworld:latest --image-pull-policy=Never --port=80
+kubectl port-forward pods/hello-world 8080:80
 ```
-  </TabItem>
-</Tabs>
 
-Point your web browser to `localhost:8080`, and you will see the nginx intro screen. If you prefer to stay on the command line, use `curl localhost:8080`.
+Point your web browser to `localhost:8080`, and you will see the message `Hello World from NGINX!!`. If you prefer to stay on the command line, use `curl localhost:8080`.
 
-**To stop, remove the container and delete the image:**
+**To delete the pod and the image:**
 
 <Tabs groupId="container-runtime">
   <TabItem value="nerdctl" default>
 
 ```
-nerdctl stop <container-id>
-nerdctl rm <container-id>
-nerdctl ps #to verify nothing is running 
-nerdctl rmi nginx-helloworld:v1.0 #to remove the image
+kubectl delete pod hello-world
+nerdctl rmi nginx-helloworld:latest --namespace k8s.io #to remove the image
```

Comment:
The `--namespace` option must come **before** `rmi`:

```suggestion
# remove the image
nerdctl --namespace k8s.io rmi nginx-helloworld:latest
```

Reply:
Modified per your suggestion.